editor.planx.uk/src/@planx/components/Feedback/Public.tsx

```diff
@@ -0,0 +1,119 @@
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Grid";
+import Link from "@mui/material/Link";
+import Typography from "@mui/material/Typography";
+import Card from "@planx/components/shared/Preview/Card";
+import { CardHeader } from "@planx/components/shared/Preview/CardHeader/CardHeader";
+import type { PublicProps } from "@planx/components/ui";
+import { useFormik } from "formik";
+import React from "react";
+import RichTextInput from "ui/editor/RichTextInput/RichTextInput";
+import TerribleFace from "ui/images/feedback_filled-01.svg";
+import PoorFace from "ui/images/feedback_filled-02.svg";
+import NeutralFace from "ui/images/feedback_filled-03.svg";
+import GoodFace from "ui/images/feedback_filled-04.svg";
+import ExcellentFace from "ui/images/feedback_filled-05.svg";
+import InputLabel from "ui/public/InputLabel";
+
+import { getPreviouslySubmittedData, makeData } from "../shared/utils";
+import { FaceBox } from "./components/FaceBox";
+import { StyledToggleButtonGroup } from "./styled";
+import { FeedbackComponentProps, FormProps } from "./types";
+
+const FeedbackComponent = (
+  props: PublicProps<FeedbackComponentProps>,
+): FCReturn => {
+  const formik = useFormik<FormProps>({
+    initialValues: getPreviouslySubmittedData(props) ?? {
+      feedbackScore: "",
+      feedback: "",
+    },
+    onSubmit: (values) => {
+      props.handleSubmit?.(makeData(props, values));
+    },
+  });
+
+  const handleFeedbackChange = (
+    event: React.MouseEvent<HTMLElement>,
+    newValue: string | null,
+  ) => {
+    if (newValue !== null) {
+      formik.setFieldValue("feedbackScore", newValue);
+    }
+  };
+
+  return (
+    <Card handleSubmit={formik.handleSubmit}>
+      <CardHeader title={props.title} />
+      <Typography pt={4}>
+        This service is a work in progress, any feedback you share about your
+        experience will help us to improve it.
+      </Typography>
+      <Typography pt={2}>
+        Don't share any personal or financial information in your feedback. If
+        you do we will act according to our{" "}
+        <Link href={props.privacyPolicyLink ?? ""}>privacy policy</Link>.
+      </Typography>
+
+      <Box pt={2} mb={3}>
+        <InputLabel label="How would you rate your experience with this service?" />
```

Comment:
The hover issue was caused by having more than one child inside the label component, as discussed in this issue: https://github.com/mui/material-ui/issues/14543

I've made the `children` prop optional in the `InputLabel` component to get round this.

Reply:
Ah nice one 👍 

Alternatively, I _think_ we could make children a `ReactElement` which would force a single element or fragment here. This will likely have cascading type issues but may point at other places where we're hitting a similar hover issue?

The a11y tests are still passing in CI and Storybook without the wrapping label, so this is all good as-is.